### DISCUSSION TITLE
Error Loading Remote Images with Next.js Image Component

### DISCUSSION THREAD

**Discussion by MarwanEA** [community/community#135707] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to load remote images in my Next.js application using the `next/image` component, but I'm running into an issue. The error message I'm getting is:

```
Error: Invalid src prop (...) on `next/image`, hostname "www.thecocktaildb.com" is not configured under images in your `next.config.js`
```

Here’s a screenshot of the error:
![e3851c50-8676-4d61-b0d0-6703ae94678f](https://github.com/user-attachments/assets/b75dd047-2dde-486f-80af-ffe3e88a2071)

How to load images from a remote website using the Next.js?

**Comment by Achraftraf** [comment, marked as answer]
make sure to configure it in next.config.mjs like that for example

```
/** @type {import('next').NextConfig} */
const nextConfig = {
    images: {
      remotePatterns: [
        {
          protocol: 'https',
          hostname: 'URL_WEBSITE',
          port: '',
          pathname: '/images/**',
        },
      ],
    },
  };
  
  export default nextConfig;
```
